testUtils/prismaSpanBuilder.js

```diff
@@ -0,0 +1,88 @@
+import { PRISMA_SPAN } from '../src/spans/awsSpan';
+import { HttpSpanBuilder } from './httpSpanBuilder';
+import { payloadStringify } from '@lumigo/node-core/lib/common';
+
+export class PrismaSpanBuilder {
+  constructor() {
+    this._span = {
```

Comment:
you copied a function_span, so it contains all kinds of fields that you real http_span will not have (e.g. readiness)

Reply:
Got you... I copied `MongoSpanBuilder` which had all this :( I'll remove all the irrelevant fields

Reply:
Wait, I actually don't get it :) if the mongo-span has all those attributes too (e.g. `readiness`), is it by accident and those get ignored in the ingestion, or am I missing something here?

Reply:
are you sure that the mongo-span has these attributes in production as well, or is it there only in tests?
It makes no sense to have `readiness` on an HTTP span. So if it's there, then this is a bug (and then don't fix it now - just open a ticket :) )

Reply:
The mongo-span uses [getBasicChildSpan](https://github.com/lumigo-io/lumigo-node/blob/2475be801a8db3eda04c23ce97b2125bff05a01b/src/spans/mongoDbSpan.ts#L12) which in turn uses [getBasicSpan](https://github.com/lumigo-io/lumigo-node/blob/2475be801a8db3eda04c23ce97b2125bff05a01b/src/spans/awsSpan.ts#L297), returning a bunch of attributes (such as [readiness](https://github.com/lumigo-io/lumigo-node/blob/2475be801a8db3eda04c23ce97b2125bff05a01b/src/spans/awsSpan.ts#L116)) that are probably pointless in that context, so you're definitely right :) 
I'll open a ticket for cleaning this up.

Reply:
awesome 💪